app/assets/json/translations/react/en.json

```diff
@@ -859,7 +859,7 @@
   "listings.householdMaximumIncome": "Household Maximum Income",
   "listings.housingProgram": "Housing Program",
   "listings.importantProgramRules": "Important Program Rules",
-  "listings.includesPriorityUnits": "Includes Priority Units for:",
+  "listings.includesPriorityUnits": "Includes Priority Units for %{priorities}",
```

Comment:
Same thing here, this ticket removed the `%{priorities}`: https://sfgovdt.jira.com/browse/DAH-1070